tests/testExecutables.py

```diff
@@ -0,0 +1,12 @@
+import unittest
+import lsst.utils.tests
+
+
+class UtilsBinaryTester(lsst.utils.tests.ExecutablesTestCase):
+    pass
+
+EXECUTABLES = None
+UtilsBinaryTester.create_executable_tests(__file__, EXECUTABLES)
```

Comment:
This was all dealt with in the SQR-012 work. If `EXECUTABLES` is `None` the system discovers the executables on its own. If you give a list it will just run those executables. Are you saying it should have been using an optional keyword arg, something like:
```python
create_executable_tests(__file__, exes=EXECUTABLES)
```
?